index.html

```diff
@@ -1,14 +1,259 @@
+          some of the common signs that your Liver needs help.</p>
+        <section class="line-break">
+          <div class="double-line"></div>
+        </section>
+        <br>
+        <br>
+        <a href="/blog-post.html">
+          <button class="blog-post">July 28</button></a>
+        <h5>Eat Naturally - As Close to Source as Possible</h5>
+        <p>Unlike many other diets, the Ayurvedic diet provides personalized recommendations about which foods to eat
+          and avoid based on your body type!</p>
+
+        <section class="line-break">
+          <div class="double-line"></div>
+        </section>
+        <br>
+        <br>
+        <a href="/blog-post.html">
+          <button class="blog-post">July 24</button></a>
+        <h5>Choose Fruits and Vegetables</h5>
+        <p>Eat loads of fruits and vegetables, not only for their nutritional value, but also because they are good
+          natural internal cleansers. The specific food guidelines found above for Vata, Pitta and Kapha can help us
+          pick a variety of fruits and vegetables suited to our physiology and the season. Vegetables do not necessarily
+          have to be just separate dishes. Add them to grains, mix them with beans, toss them in stews and soups —
+          there's always room for your favorite veggies in every dish. .</p>
+        <section class="line-break">
+          <div class="double-line"></div>
+        </section>
+        <section class="line-break">
+          <div class="double-line"></div>
+        </section>
+        <br>
+        <br>
+      </section>
+      <!-----Blog post with pictures wrap----->
+      <section id="bottom-blog">
+        <h4>Cleanse from the inside out</h4>
+        <div id='image-blog'><img src='\images\gulabjamooncake.jpg' alt="Healthy" width="75" height="75" /></div>
+        <p>Completing a cleanse during every change of seasons, to detox and rid the body of ama, is recommended for
+          optimal health</p>
+        <a href="/learn-more.html">
+          <button class="learn-more">Learn more</button><br></a>
+        <section class="line-break">
+          <div class="double-line"></div>
+        </section>
+
+        <h4>Drink to your health!</h4>
+        <div id='image-blog'><img src='\images\katiroll.jpg' alt="Healthy" width="75" height="75" /></div>
+        <p>When possible avoid caffeine and alcohol and carbonated soft drinks, and switch to life-giving,
+          vitality-boosting beverages. RASA offers a whole host of SuperDrinks that will feed body, heart and mind.
+          Start with water, that most basic yet most overlooked drink — drinking lots of warm water through the day
+          helps to rehydrate our system and flush toxins out of the body.</p>
+        <a href="/learn-more.html">
+          <button class="learn-more">Learn more</button></a>
+
+      </section>
+  </section>
+  <!-------Footer-------->
+  <section id="footer">

```

Comment:
Making this `section` the width of the page and pushing the background to the edges would fit the project description a little bit better.